wire-ios-data-model/Source/MLS/MLSService.swift

```diff
@@ -221,10 +223,13 @@ public final class MLSService: MLSServiceInterface {
             subconversationGroupIDRepository: subconversationGroupIDRepository
         )
 
-        // FIXME: [jacob] fetch on demand when creating group
-        fetchBackendPublicKeys()
-
         schedulePeriodicKeyMaterialUpdateCheck()
+
+        // FIXME: [jacob] fetch on demand when creating group
+        Task {
+            await fetchBackendPublicKeys()
+            delegate?.MLSServiceDidFinishInitialization()
```

Comment:
@typfel  I had to keep this new delegate function for now, since `fetchBackendPublicKeys` became async.

Reply:
OK, I'll remove it when I move `fetchBackendPublicKeys` into the group creation flow